connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/executable/InboundExecutableRegistryImpl.java

```diff
@@ -311,6 +288,13 @@ private ActiveExecutableResponse mapToResponse(UUID id, RegisteredExecutable con
                   new Error(
                       "Activation failure", "Invalid connector definition: " + invalid.reason())),
               List.of());
+      case Cancelled cancelled ->
+          new ActiveExecutableResponse(
+              id,
+              cancelled.executable().getClass(),
+              cancelled.context().connectorElements(),
+              cancelled.context().getHealth(),
+              cancelled.context().getLogs());
```

Comment:
Do we report the exception somewhere that cause the cancellation? Would be enough if its part of the health object.

Reply:
true 👍